src/api-v1.js

```diff
@@ -665,13 +666,17 @@ api.declare({
       token: token,
     });
 
+    let workerId = slugid.v4();
+    let workerGroup = 'ec2_' + secret.region;
```

Comment:
So it looks like we're changing workerGroup to be ec2_<region> instead of just <region> which is ok as far as I'm concerned.  I would like us to maintain region somehow because we use that in metrics and logs for knowing which region a particular set of instances came from.